Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Hole_filling/Triangulate_hole_polyline.h

```diff
@@ -1279,37 +1341,17 @@ triangulate_hole_polyline_with_cdt(const PointRange1& points,
   if (num_normals < 1)
     return false;
 
-  // Find non-collinear the hole boundary points in order
-  // to check the hole orientation.
-  std::vector<Point_3> pts;
-  pts.reserve(3);
-  pts.push_back(ref_point);
-  for (std::size_t i = 1; i < size; ++i) {
-    const std::size_t ip = i + 1;
-    const Point_3& p1 = ref_point;
-    const Point_3& p2 = P[i];
-    const Point_3& p3 = P[ip];
-    if (!collinear_3(p1, p2, p3)) {
-      pts.push_back(p2);
-      pts.push_back(p3);
-      break;
-    }
-  }
-
-  // Not sure about that!
-  if (CGAL::orientation(pts[0], pts[1], pts[2], Q[0]) != CGAL::NEGATIVE) {
-    // std::cout << "positive" << std::endl;
-  } else {
-    // std::cout << "negative" << std::endl;
-    x = -x; y = -y; z = -z;
-  }
-
   // Setting the final normal.
   x /= static_cast<FT>(num_normals);
   y /= static_cast<FT>(num_normals);
   z /= static_cast<FT>(num_normals);
   const Vector_3 avg_normal = Vector_3(x, y, z);
-  // std::cout << "avg: " << avg_normal << std::endl;
+  // std::cout << "avg normal: " << avg_normal << std::endl;
+
+  // Checking the hole planarity.
+  const FT max_distance = FT(2) / FT(100);
```

Comment:
Is this a constant that does not depend on the input?

Reply:
It does, but as I wrote in the comment above I have not yet added it to the input, because I would first like to get opinions on whether we need this planarity test at all. If we do not, then we do not need this constant either.